trade-finance-manager-ui/server/controllers/helpers/get-success-banner-message.helper.ts

```diff
@@ -9,9 +9,15 @@ import { isDealCancellationEnabled } from './deal-cancellation-enabled.helper';
  * @param userToken - the user token
  * @returns the success message to be shown or null
  */
-export const getScheduledCancellationBannerMessage = async ({ dealSnapshot, userToken }: { dealSnapshot: Deal; userToken: string }): Promise<string | null> => {
+export const getScheduledCancellationBannerMessage = async ({
+  dealSnapshot,
+  userToken,
+}: {
+  dealSnapshot: MappedDealSnapshot;
+  userToken: string;
+}): Promise<string | null> => {
   const { submissionType, _id: dealId } = dealSnapshot;
-  const ukefDealId = getUkefDealId(dealSnapshot);
+  const { ukefDealId } = dealSnapshot.details;
```

Comment:
In the mapped snapshot, the `ukefDealId` exists in the same place on all deal snapshot types (in `details.ukefDealId`)